server/src/server.rs

```diff
@@ -138,6 +141,14 @@ impl Server {
                 .ok_or(NotFound::NodeBySlot(slot))?;
             (src_node, dest_node)
         };
+        if let Err(e) = src_node
+            .forwarding_limiter
+            .check_n(NonZeroU32::new(packet.payload.len().try_into().unwrap_or(u32::MAX)).unwrap())
+        {
+            log::trace!("Rate limiting: {:?}", e);
+            // TODO: Send `PauseForwarding`
```

Comment:
What would you say @mfranciszkiewicz ?

Reply:
We probably cannot merge rate limiting alone since it might simply cut off communication at some point. We need both `[Pause|Resume]Forwarding` and backpressure handling within the Client

Reply:
Will be done in https://github.com/golemfactory/ya-relay/issues/42